.gitignore

```diff
@@ -32,6 +32,9 @@ package-lock.json
 # build
 src/version.ts
 
+# IDEs
+.idea/
+
```

Comment:
IDE specific files should not go in the .gitignore. The Daffodil .gitignore file has a comment explaining the reasoning.